.github/workflows/container-image.yml

```diff
@@ -46,7 +46,17 @@ jobs:
       uses: ishworkh/docker-image-artifact-upload@v1
       with:
         image: "samba-server:latest"
+
         retention_days: 1
+  build-server-centos:
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+      uses: actions/checkout@v3
+      name: Build the CentOS based server image
+      rin: male OS_NAME=centos build-server
```

Comment:
What is `rin`? What is `male`? -- typos?

Reply:
Pretty sure those are typos. Interestingly, they seem to have prevented the CI jobs from starting.

Reply:
@synarete  wrote:
> What is `rin`? What is `male`? -- typos?


yes, typos: `rin`  is `run` and `male` is `make`. will push a fixed update ...

Reply:
fixed